x-pack/plugins/endpoint/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I would prefer if we renamed this configuration file so that it's not picked up automatically by IDEs as the configuration for the whole endpoint app code base. You could rename it to something like `cli_tsconfig.json` and then point `ts-node` to use it like:
```
ts-node --project cli_tsconfig.json scripts/resolver_generator.ts
```